.gitignore

```diff
@@ -19,3 +19,7 @@ MANIFEST-*
 OPTIONS-*
 CURRENT
 LOCK
+
+# standard file naming when generating benchmarks through "go ppof"
```

Comment:
nit: I think the file naming is a convention rather than requirement, and `*.prof` is more commonly used and also used in the tutorial ([source](https://go.dev/blog/pprof))